requirements/base.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.11.5
+Django==1.11.15
```

Comment:
Could change to `Django>=1.11,<1.12` so you don't have to keep updating it for security patches, if you want. But otherwise :+1: